Public/New-HTMLStatusItem.ps1

```diff
@@ -1,43 +1,72 @@
 function New-HTMLStatusItem {
-    [CmdletBinding()]
+    [CmdletBinding(DefaultParameterSetName = 'Percent')]
     param(
         [string] $ServiceName,
+
         [string] $ServiceStatus,
-        [ValidateSet('Dead', 'Bad', 'Good')]$Icon = 'Good',
-        [ValidateSet('0%', '10%', '30%', '70%', '100%')][string] $Percentage = '100%'
+
+        [ValidateSet('Dead', 'Bad', 'Good')]
+        [string]$Icon = 'Good',
+
+        [ValidateSet('0%', '10%', '30%', '70%', '100%')]
+        [Parameter(ParameterSetName = 'Percent')]
+        [string]$Percentage = '100%',
+
+        [ValidatePattern('^#([A-Fa-f0-9]{6}|[A-Fa-f0-9]{3})$')]
+        [Parameter(ParameterSetName = 'Hex')]
```

Comment:
For colors you should use what other commands are using. Register-ArgumentCompleter. It allows for pick from 800 colors but also takes '#COLOR' if typed in.